src/app/Http/Controllers/Operations/Concerns/HasForm.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace Backpack\CRUD\app\Http\Controllers\Operations\Concerns;
+
+use Illuminate\Support\Facades\Route;
+use Illuminate\Support\Str;
+
+trait HasForm
+{
+    protected function formRoutes(string $operationName, bool $routesHaveIdSegment, string $segment, string $routeName, string $controller): void
```

Comment:
Once HasForm is used, this will be a method on the CrudController... so maybe the name is too general? Might conflict with custom methods.